src/geometry/Transformation.js

```diff
@@ -9,21 +9,33 @@
  * @example
  *
  * ```js
- * var transformation = new L.Transformation(2, 5, -1, 10),
+ * var transformation = L.transformation(2, 5, -1, 10),
  * 	p = L.point(1, 2),
  * 	p2 = transformation.transform(p), //  L.point(7, 8)
  * 	p3 = transformation.untransform(p2); //  L.point(1, 2)
  * ```
  */
 
 
-// factory new L.Transformation(a: Number, b: Number, c: Number, d: Number)
+// factory new L.Transformation(a: Number|Number[], b?: Number, c?: Number, d?: Number)
 // Creates a `Transformation` object with the given coefficients.
 L.Transformation = function (a, b, c, d) {
-	this._a = a;
-	this._b = b;
-	this._c = c;
-	this._d = d;
+	// very conservative way to allow a only. not sure if needed.
+  if (L.Util.isArray(a) && a.length === 4 && !b && !c && !d) {

```

Comment:
What about "a.length === 4"? If a.length <4, index-errors may appear which did not appear with the current state of the code.